com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -367,6 +367,11 @@ public class NetworkSceneManager : IDisposable
         /// </summary>
         internal Dictionary<int, uint> BuildIndexToHash = new Dictionary<int, uint>();
 
+        // For full scene transitions we want to keep track of in-scene placed NetworkObjects' active state that are moved into the DDOL
+        // by user code.
+        private Dictionary<ulong, bool> m_NetworkObjectDDOLActiveState = new Dictionary<ulong, bool>();
```

Comment:
Can you add a comment stating what the key is ? (I guess the NetworkObjectId)

Reply:
Removing the state tracking and leaving it up to the user as to whether they want to disable during full scene transition or not.